### ISSUE TITLE
Adding Stream Data to GLOS?

### ISSUE THREAD

**Issue by beckypearson** [glos/user_queries#6]
Russel Tooley, 

Emailed info@glos.us this: 
Greetings... We do stream monitoring in 5 Manitowoc County creeks that empty into Lake Michigan and would like to give data access to GLOS. How do we do that? Thank You

Becky's response thus far: 

Thanks for contacting us. We have a data management team that will work with you to make your data discoverable and publicly accessible. To begin, could you tells us a little bit more about your data? 
Do you do real-time monitoring? 
What parameters to do you monitor? 
How far back in years do the data go? 
I assume that you will be hosting your data on a serving that you can can push to GLOS. Does that sound right? 

I am copying Kathy, our data curator, since she will help document the data in our metadata catalog. Additionally, we need to assess the effort/resources available to get your data into our portal. Than we can work on a timeline.

**Comment by kknee** [comment]
@beckypearson can we request a sample of the data?  That will help us gauge level of effort for integration.

**Comment by beckypearson** [comment]
Yes, I will ask him right now for a sample. Stay tuned.

**Comment by kkoch** [comment]
Yes and another question you can start asking users right up front I think would be what quality controls they have in place (just so they aren't surprised that we want that).  Once we get details back from him then eithe @kknee or myself can create a github ticket over on the dmac side to investigate in more detail.
